src/serde_helpers.rs

```diff
@@ -127,7 +181,9 @@ macro_rules! impl_deserialize_for_internally_tagged_enum {
         $tag:literal,
         $(
             ($variant_tag:literal => $($variant:tt)+ )
-        ),* $(,)?
+        ),*
+        $(, (_ => $($default_variant:tt)+ ))?
+        $(,)*
```

Comment:
```suggestion
        $(, (_ => $($default_variant:tt)? ))?
        $(,)?
```
to allow only one default variant and do not allow multiple trailing commas